src/main/java/core/basesyntax/service/impl/ReportGenerateServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerateService;
+import java.util.Map;
+
+public class ReportGenerateServiceImpl implements ReportGenerateService {
```

Comment:
remember about the correct naming for classes
```suggestion
public class ReportGeneratorServiceImpl implements ReportGeneratorService {
```

Reply:
ok